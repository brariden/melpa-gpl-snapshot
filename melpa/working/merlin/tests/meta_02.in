#!/bin/bash
. functions.sh || exit 1
reset meta_02.ml

# Without meta, ( >. ) is a valid operator
tell <<END
let ( >. ) : float -> float -> bool = ( > );;

1.0 >. 0.0;;
END

cmd errors
cmd dump env
