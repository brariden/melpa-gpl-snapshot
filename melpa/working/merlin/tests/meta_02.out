["return",{"cursor":{"line":1,"col":0},"marker":false}]
["return",{"cursor":{"line":1,"col":0},"marker":false}]
["return",{"cursor":{"line":4,"col":0},"marker":false}]
["return",{"cursor":{"line":4,"col":0},"marker":false}]
["return",[]]
["return",[["File \"meta_02.ml\", line 1, characters 4-10","val ( >. ) : float -> float -> bool"]]]
